litho-core/src/main/java/com/facebook/litho/InternalNode.java

```diff
@@ -1446,8 +1522,17 @@ InternalNode removeChildAt(int index) {
     return (InternalNode) mYogaNode.removeChildAt(index).getData();
   }
 
+
+  private void checkIfBuilt() {
```

Comment:
Can we rename it to something like `checkIfMutable()`? Because `checkIfBuilt()` reflects the method name rather than the idea behind it. What we want to avoid is mutating an object that shouldn't be mutable anymore.